### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by thepian** [socketstream/documentation-gitbook#2]
I think this is a good approach if it is more of a story based introduction type thing. The nice thing is easy linking and the ability to fork it.

Did you consider the ability to treat the Wiki as a git REPO. We could have the wiki as a submodule to socket stream and a build step for deploying documentation on socket stream.org

The two could co-exist with the Wiki being a FAQ and this being more of a tutorial.

I think the djangoproject.org has a lot of good documentation approaches including the django book.

**Comment by arxpoetica** [comment]
Totally. That's exactly the aim.

**Comment by arxpoetica** [comment]
Yes and no. I think "tutorial" is the wrong word. Good documentation reads like a tutorial—and we've never had full on "spell it out" documentation of that kind. It probably won't suffice to try and explain it over GitHub. We should talk about this more direct if possible. You can DM if you want: https://twitter.com/arxpoetica and I can try and set something up.
